scalyr_agent/scalyr_client.py

```diff
@@ -312,7 +312,9 @@ def __send_request(
             else:
                 body_str = ""
 
-            body_str_raw = body_str
+            # Workaround to fix issue with logging non utf-8 characters. We simply ignore
+            # non utf-8 characters
+            body_str_raw = body_str.decode("utf-8", "ignore")
```

Comment:
The issue is related to ``LogRecord.getMessage`` throwing because the actual request body can also contain non utf-8 characters:

```bash
Traceback (most recent call last):
  File "/usr/lib/python2.7/logging/handlers.py", line 76, in emit
    if self.shouldRollover(record):
  File "/usr/lib/python2.7/logging/handlers.py", line 156, in shouldRollover
    msg = "%s\n" % self.format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 734, in format
    return fmt.format(record)
  File "/home/vagrant/w/scalyr/scalyr-agent-2/scalyr_agent/scalyr_logging.py", line 852, in format
    return BaseFormatter.format(self, record)
  File "/home/vagrant/w/scalyr/scalyr-agent-2/scalyr_agent/scalyr_logging.py", line 802, in format
    result = logging.Formatter.format(self, record)
  File "/usr/lib/python2.7/logging/__init__.py", line 465, in format
    record.message = record.getMessage()
  File "/usr/lib/python2.7/logging/__init__.py", line 329, in getMessage
    msg = msg % self.args
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc0 in position 316: ordinal not in range(128)
```